ruby/.rubocop-1-60-except-graphql.yml

```diff
@@ -171,6 +171,13 @@ RSpec/NamedSubject:
 RSpec/NestedGroups:
   Enabled: false
 
+# for rubocop-capybara
+# Documentation can be found at https://docs.rubocop.org/rubocop-capybara/cops_capybara.html#capybaraclicklinkorbuttonstyle
+Capybara/ClickLinkOrButtonStyle:
+  EnforcedStyle: link_or_button
```

Comment:
Since `link_or_button` is the default on [ClickLinkOrButtonStyle](https://docs.rubocop.org/rubocop-capybara/cops_capybara.html#capybaraclicklinkorbuttonstyle), do we need to add the EnforceStyle here?

Reply:
See my commit message 😄
![image](https://github.com/q-centrix/style-guides/assets/11759443/e207bad0-7de0-4281-833a-f662386d7a7a)

Reply:
I could add a comment to remove it in the next rubocop styleguide version update?

Reply:
That makes sense. I think I'm okay without the comment.